examples/2016-10-31/api_backend/template.yaml

```diff
@@ -7,7 +7,7 @@ Resources:
     Properties:
       Handler: index.get
       Runtime: nodejs6.10
-      CodeUri: s3://<bucket>/api_backend.zip
+      CodeUri: src/
```

Comment:
I am assuming you tested that these work with the `package` and `deploy` commands?

Could you explain how you verified? Did you deploy and package from the repo or pull them our of here and deploy/package that way?

Reply:
I want to add my support for this PR. When first learning the tools, I modified these files to have:
```
CodeUri: ./
```
...in order to use the `sam local package` and `sam local deploy` commands for deploying the examples. Having an S3 link in the example template adds a complexity that makes it more difficult to follow the examples without prior experience. I like what @heitorlessa has done here.

Reply:
The other nice thing about this usage of `CodeUri` is that it will work out-of-the-box with the SAM Local tools, such as `sam local start-api`

Reply:
@jfuss I packaged and deployed each of those when testing them after these changes. Also made me notice that encryption_proxy had a bug as KMS changed the return to Bytes and there wasn't any decode in there -- Added an extra commit to fix that too.

I'll try to create a separate PR to include README with a given format that we can discuss as I agree with that idea too

Reply:
AWESOME!!!!